### PR TITLE
Fix #420, Sync OBU as indication of no parameter blocks

### LINKED ISSUE #420: Consider Sync OBU as indication to use default value of a parameter
Descriptor OBUs are describing `audio_substream_id` and `parameter_id` and Sync OBU can indicate its presence until the next Sync OBU comes up.
When we consider #416, Sync OBU may be used to indicate the situation that device needs to apply default value of a parameter by not storing the parameter_id which is defined in Descriptor OBUs.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
Updated on Sync OBU syntax and semantics section.

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/432.html" title="Last updated on May 26, 2023, 5:24 PM UTC (a9e2a38)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/432/2c7aa61...a9e2a38.html" title="Last updated on May 26, 2023, 5:24 PM UTC (a9e2a38)">Diff</a>

### CODE REVIEW

**Review by sunghee-hwang — COMMENTED** [review]
For sync obu improvement (editorial)

File: index.bs [hunk 1]
```diff
@@ -1530,6 +1530,8 @@ NOTE: The Temporal Delimiter OBU has an empty payload.
 
 This section specifies the OBU payload of OBU_IA_Sync.
 
+Any identifier which is not specificed in the preceding Descriptor OBUs, shall not be present in this OBU. When a [=parameter_id=] specified in the preceding Descriptor OBUs is not defined in this OBU, it indicates that no Parameter Block OBUs with the [=parameter_id=] are provided until the next Sync OBU.
```
Comment (cconcolato) [inline index.bs:1533]: Can you move this paragraph in the semantics of the identifiers? 
Can we use a shall statement in the second sentence? For example: 
```
For every Parameter Block OBU (resp. Audio Frame OBU) in an IA Sequence, the value of its parameter_id (resp. audio_substream_id) shall be listed as value of obu_id in the preceding Sync OBU.

**Review by tdaede — APPROVED** [review]

**Comment by tdaede** [comment]
We will merge this and create a new PR to remove the sync OBU from this version.

**Comment by tdaede** [comment]
(this needs a rebase before merge)

**Pull request merged**